test/files/neg/override-concrete-type.check

```diff
@@ -0,0 +1,5 @@
+override-concrete-type.scala:7: error: overriding type A1 in class Foo, which equals Something;
+ overriding concrete type alias type A1 is not allowed except when equivalent
```

Comment:
Is one-space indentation intentional? I can't say that ten times fast.

Reply:
I see there is an existing one-space indent.

Reply:
I am happy to change this to two space indent.